pkg/vm/vm.go

```diff
@@ -311,16 +320,31 @@ func (v *VM) execute(ctx *Context, op Instruction) {
 		l := int(v.estack.Pop().BigInt().Int64())
 		o := int(v.estack.Pop().BigInt().Int64())
 		s := v.estack.Pop().Bytes()
+		if l < 0 {
+			panic("negative index")
```

Comment:
shouldn't it be the other way? (here `negative length`, 3 lines below -- `negative index`)

Reply:
Yep, thanks. Will fix.